torch/xpu/__init__.py

```diff
@@ -0,0 +1,199 @@
+from functools import lru_cache
```

Comment:
Can you please add some general comments about this package  ? Please refer to cuda, mps __init__.py files

Reply:
Done.